src/Favicon/Favicon.php

```diff
@@ -311,9 +314,13 @@ private function checkImageMType($url) {
     private function checkImageMTypeContent($content) {
         if(empty($content)) return false;
 
-        $fInfo = finfo_open(FILEINFO_MIME_TYPE);
-        $isImage = strpos(finfo_buffer($fInfo, $content), 'image') !== false;
-        finfo_close($fInfo);
+        $isImage = true;
+        try {
+            $fInfo = finfo_open(FILEINFO_MIME_TYPE);
+            $isImage = strpos(finfo_buffer($fInfo, $content), 'image') !== false;
+            finfo_close($fInfo);
+        } catch (Exception $e) {
+        }
```

Comment:
I'm not sure that `finfo_*` will ever raise an exception, but at the very least the error should be logged using `error_log`.